schema/crdb/README.adoc

```diff
@@ -68,20 +85,21 @@ Process:
 SQL Validation, via Automated Tests:
 
 * The `SCHEMA_VERSION` matches the version used in `dbinit.sql`
-* The combination of all `up.sql` files results in the same schema as `dbinit.sql`
+* The combination of all `up.sql` files results in the same schema as
+  `dbinit.sql`
 * All `up.sql` files can be applied twice without error
 
-==== Handling common schema changes
+==== General notes
 
-Although CockroachDB's schema includes some opaque internally-generated fields
-that are order dependent - such as the names of anonymous CHECK constraints -
-our schema comparison tools intentionally ignore these values. As a result,
-when performing schema changes, the order of new tables and constraints should
-generally not be important.
+CockroachDB's representation of the schema includes some opaque
+internally-generated fields that are order dependent, like the names of
+anonymous CHECK constraints.  Our schema comparison tools intentionally ignore
+these values. As a result, when performing schema changes, the order of new
+tables and constraints should generally not be important.
 
-As convention, however, we recommend keeping the `db_metadata` file at the end of
-`dbinit.sql`, so that the database does not contain a version until it is fully
-populated.
+As convention, however, we recommend keeping the `db_metadata` file at the end
+of `dbinit.sql`, so that the database does not contain a version until it is
+fully populated.
 
 ==== Adding new source tables to an existing view
```

Comment:
This section threw me off.  What's "the correct position" (i.e., position in what order? why does it matter?)  Is this still relevant after #4288?  If it is, maybe it'd help to move this section and the next one under a common heading like "Gotchas around specific kinds of schema changes" so that it's clearer that these are unusual cases and you can skip it altogether if you're obviously not in one of them.

Reply:
I think this is a good question for @gjcolombo -- this was added in 876e8ca7c6 with some of the instance work.

I **think** this is obsolete now that ordering no longer matters, as you say, after #4288.

Reply:
I concur--I think the "Adding new source tables to an existing view" section is obsoleted by #4288 and can be removed.

Reply:
Thanks!  Done in [51fcf64](https://github.com/oxidecomputer/omicron/pull/4393/commits/51fcf6476f978b851fa594ebd18623287627f6da).